app/views/shared/_user_nav.haml

```diff
@@ -1,5 +1,5 @@
-.user_nav
-  - if current_user
+- if current_user
+  .user_nav
```

Comment:
Not for this nav, no. It only exists for reviewers and admins; there's not a login functionality for folks submitting proposals.